_posts/en/newsletters/2023-11-01-newsletter.md

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Bitcoin Optech Newsletter #275'
+permalink: /en/newsletters/2023/11/01/
+name: 2023-11-01-newsletter
+slug: 2023-11-01-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Continued discussion about scripting changes:** there were several
+  replies on the Bitcoin-Dev mailing list to discussions we've
+  previously covered.
+
+    - *Covenants research:* Anthony Towns [replied][towns cov] to a
+      [post][russell cov] by Rusty Russell that we mentioned [last
+      week][news274 cov].  Towns compares Russell's approach to other
+      approaches specifically for [covenant][topic covenants]-based
+      [vaults][topic vaults] and finds it unappealing.  In a further
+      [reply][russell cov2], Russell notes that there different designs
+      for vaults and that vaults are fundamentally less optimal than
+      other transactions types, implying that optimization isn't
+      critical for vault users.  He argued that the [BIP345][] vault
```

Comment:
BIP345 is not merged yet, so link to https://github.com/bitcoin/bips/pull/1421 instead?